### PR TITLE
feat: create the function to check if a given number is perfect or not

### PR DESCRIPTION
**Pull request by nammon72**
1. What did you do? (mandatory)
- Create the function to check if a given number is perfect or not
2. Why would you do that? (mandatory)
- Use for-loop to find all divisors and add them
- Declare the "sum" variable to store the sum of divisors
- If the sum of divisors is equal to n, then n is a perfect number
3. An explanation about your `solution or decision or choise` (if needed)

4. Checklist (if needed)

5. Action need to do before merge (if needed)

6. Action need to do after merge (if needed)

7. Others note (if needed)

### CODE REVIEW

File: src/perfect-number.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function isPerfect(n) {
+  let sum = 0;
+  for (let i = 1; i <= n / 2; i += 1) {
+    if (n % i === 0) sum += i;
```
Comment (anhdiepmmk) [inline src/perfect-number.js:4]: n % !==0 then return false

File: src/perfect-number.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  }
+  if (sum === n) return true;
```
Comment (anhdiepmmk) [inline src/perfect-number.js:6]: remove lines 6,7. return true instead

File: src/perfect-number.js (hunk 1 above)
Comment (nammon72) [reply src/perfect-number.js:4]: no, it needs to find all divisors and add them.

File: src/perfect-number.js (hunk 2 above)
Comment (nammon72) [reply src/perfect-number.js:6]: If sum of divisors is equal to n, then n is a perfect number.

**Review by anhdiepmmk — APPROVED** [review]

**Pull request merged**